OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -47,13 +48,19 @@ public IEnumerable<IActorPreview> RenderPreviewSprites(ActorPreviewInitializer i
 				p = init.WorldRenderer.Palette(Palette);
 
 			var body = init.Actor.TraitInfo<BodyOrientationInfo>();
-			var facing = init.Contains<FacingInit>() ? init.Get<FacingInit, int>() : 0;
-			var anim = new Animation(init.World, image, () => facing);
+			Func<int> facing = init.Contains<FacingInit>() ? (Func<int>)(() => init.Get<FacingInit, int>()) : () => 0;
```

Comment:
I could put it in yet another helper function - it's a bit tricky to assign a lambda function using the ternary operator. (see http://stackoverflow.com/questions/11309557/assign-a-lambda-expression-using-the-conditional-ternary-operator)